File: InitializrSpringbootProject/src/main/java/com/seminario/integrador/pawplan/controller/values/FiltroTurnoRq.java
package com.seminario.integrador.pawplan.controller.values;

import java.sql.Date;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Data;

@Data
public class FiltroTurnoRq {
    @JsonProperty("idAnimal")
    private Long idAnimal;
    @JsonProperty("idCliente")
    private Long idCliente; 
    @JsonProperty("idVeterinaria")
    private Long idVeterinaria;
    @JsonProperty("idVeterinario")
    private Long idVeterinario;
    @JsonProperty("nEstado")
    private String nEstado;
    @JsonProperty("fecha")
    private Date fecha;

    @JsonProperty("page")
    private int page;
    @JsonProperty("size")
    private int size;
    @JsonProperty("orderDir")
    private String orderDir;
    @JsonProperty("orderBy")
    private String orderBy;
}

File: PawPanFront/src/app/model/FiltroTurnoRq.ts
export class FiltroTurnoRq {
    idAnimal: number //filtra solo para el cliente
    idCliente: number; //no filtra se setea por session
    idVeterinaria: number; //filtra solo para la veterinaria sino se setea por session
    idVeterinario: number; //no filtra se setea por session
    nEstado: string; //data-Estados esta el nombre igual que en bd
    fecha: string; //segun datepicker

    page: number;
    size: number;
    orderDir: string;
    orderBy: string;
}
